root/usr/share/fdi/commonfunc.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/echo This file is meant to sourced not executed:
```

Comment:
When executed this looks better :-)

aschiermeier@pf011e5k:~/Source/foreman-discovery-image> ./root/usr/share/fdi/commonfunc.sh 
This file is meant to be sourced not to be executed: ./root/usr/share/fdi/commonfunc.sh
aschiermeier@pf011e5k:~/Source/foreman-discovery-image>